.github/workflows/build.yml

```diff
@@ -56,3 +56,13 @@ jobs:
           folder: static/weaver-components
           clean: true
           single_commit: true
+
+      - name: Cleanup xvfb
```

Comment:
I do not think this is needed. Also, we can use official coverapps action. We are using the forked branch for affording json coveralls. We intend to switch back to the official action when this [PR](https://github.com/coverallsapp/github-action/pull/46) is merged. We do not need the fork for this repo.

Reply:
Removed cleanup xvfb block